src/main/java/fmartin1/model/common/NamedAPIResource.java

```diff
@@ -0,0 +1,41 @@
+package fmartin1.model.common;
+
+public class NamedAPIResource {
```

Comment:
Maybe to be more specific this class should be renamed to be: `NamedApiResourceModel` or so